### DISCUSSION TITLE
v19.12.3-3.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#111] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.12.3`  
Current YTLite Version: `3.0.1`  
Current App Compatibility: `iOS/iPadOS 14.0` or later

**RELEASE 19.12.3-3.0.1:** (2024-03-27)

- Updated Youtube from `19.10.7` to `19.12.3`
- Updated submodules
   - YTUHD 
   - YouPiP 
   - YouTubeHeader
  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*
- VersionSpoofer (Lite) doesn't work with YTLite *(as the version is already spoofed by YTLite to fix YTClassicVideoQuality)*
- Hiding Shorts creates an empty space instead #55

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.12.3-3.0.1'>v19.12.3-3.0.1 - YTLitePlus</a>.</em>

**Comment by shahin365** [comment]
@Balackburn could you please update the catbox link to the new version? it's still the old file.

**Comment by abheizthe** [comment]
some people was reporting about iSponsorBlock causing app crash/ error/ video not playing from YouTube 19.08.x - 19.12.3 version. highly recomended to install old/ lower version due tweak compatibility

**Reply by TV365-VN** [reply]
TẮT VÀ KHỞI ĐỘNG LẠI YOUTUBE TRONG PHẦN BẬT BIỂU TƯỢNG CÁI KHIÊN BẢO VỆ CHẶN QUẢN CÁO ĐỂ THAY ĐỔI CÓ HIỆU QUẢ NẾU KHÔNG THỂ CHƠI LỖI ỨNG DỤNG
==>TURN OFF AND RESTART YOUTUBE IN THE ENABLE SECTION THE AD BLOCKING PROTECTION SHIELD FOR EFFECTIVE CHANGES IF THE APP CANNOT BE PLAYED ERROR

**Reply by TV365-VN** [reply]
GO TO iSponsorBlock TURN OFF '' RESTAR YOUTUBE CHANGES TO TAKE EFFECT ''

**Comment by TV365-VN** [comment]
IF THE APPLICATION SELF-EXITS DO NOT USE IT
GO TO iSponsorBlock OFF ''REINIT YOUTUBE CHANGES TO TAKE EFFECT''

**Comment by arichornlover** [comment]
If anyone noticed that the compatibility was changed for LowContrastMode.
old: **LowContrastMode does not work (Compatible only with YouTube v16.46.5-v17.38.10)**
new: **LowContrastMode does not work (Compatible only with YouTube v17.33.2-v17.38.10)**
it basically just means this tweak only supports **v17.33.2-v17.38.10** which can be a problem though.

so @Balackburn we gotta somehow make it work correctly beyond v17.38.10

**Reply by Balackburn** [reply]
Yes, for now it's useless we should update it for latest version

**Reply by arichornlover** [reply]
The problem is, I don’t know how to make it work with the newer releases, but I’ll look around and see if it is possible to make it compatible.

**Comment by Tao1811** [comment]
excuseme what is the decrypted youtube ipa?...

**Reply by abheizthe** [reply]
i just don't know how to explain this but IMHO, it's just an app that you can install by yourself, without AppStore (sideloading)

AppStore app = encrypted. cannot install without decrypting the app
sideloading method = app from AppStore that has been decrypted 

if you need decrypted app, download from [this](https://armconverter.com/decryptedappstore) website
